src/services/admin.rs

```diff
@@ -459,17 +464,29 @@ impl AdminService {
                 ),
             };
             let mut database = self.database.lock().await;
-            node.id = database.create_node(node.clone())?;
+            dbg!("Locked the database!");
+            node.id = database.create_node(node.clone())
+                .map_err(|err| {
+                    error!("Unable to create node in Sensei Database");
+                    database.clear_pending_node();
```

Comment:
This is a "Cleaning up after myself" I mentioned before.  I got pretty good at putting my instance in this broken state.